src/lib/ipfs-utils.js

```diff
@@ -24,3 +27,9 @@ export const getIpfsCidFromUri = string => {
   }
   return ''
 }
+
+export function transformIPFSHash(str, callback) {
```

Comment:
Just wondering, are we using this?

Reply:
Yes we use it where we also use `transformAddresses `